target/linux/ramips/image/mt76x8.mk

```diff
@@ -167,6 +167,26 @@ define Device/mercury_mac1200r-v2
 endef
 TARGET_DEVICES += mercury_mac1200r-v2
 
+define Device/netgear_r6080
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_MODEL := R6080
+  DEVICE_PACKAGES := kmod-mt76x2
+  SERCOMM_HWID := CFR
+  SERCOMM_HWVER := A001
+  SERCOMM_SWVER := 0x0040
+  IMAGES += factory.img
+  IMAGE/default := append-kernel | pad-to $$$$(BLOCKSIZE)| append-rootfs | \
```

Comment:
Add a space between BLOCKSIZE and the pipe symbol (and fix that for r6120 as well, please)

Reply:
Done. I left this in its own commit for the R6120 because I wasn't sure where it would best logically fit (given the other purely cosmetic commit has already been merged). Let me know if you want me to squash it into the `ramips: move Netgear R6120 LED trigger` commit.